sc2mqtt.py

```diff
@@ -68,6 +68,14 @@ async def main():
         await ad.init()
         mqttc = mqtt.Client()
         mqttc.connect(cfo["broker"])
+        if int(cfo["brokerauth"]) == 1:
```

Comment:
It would be more convenient to remove the "brokerauth" config option, and execute authentication if "brokeruser" and "brokerpassword" are present in the config object.

Reply:
Feel free to change it as you like.